src/utils.ts

```diff
@@ -38,6 +39,37 @@ const scoreApiHeaders = {
   'Content-Type': 'application/json'
 };
 
+const DEFAULT_SCORE_API_URL = 'https://score.snapshot.org';
+
+const formatScoreAPIUrl = (url = DEFAULT_SCORE_API_URL) => {
```

Comment:
Maybe use a simple `function` for these new functions, instead of anonymous function, for better stacktrace.